### ISSUE TITLE
Broken images when parsing emails

### ISSUE THREAD

**Issue by fernandomm** [mikel/mail#1010]
I'm using this gem to parse incoming emails and detected that some images were broken. Although other email clients would show those images normally.

By broken, I mean that they can't be opened by any software ( Preview.app, Photoshop etc. ).

After further investigation I detected that the common cause are images using quoted-printable transfer encoding.

Example:

```
Content-Transfer-Encoding: quoted-printable
Content-Disposition: inline;
    filename="Logo firma.png"
Content-Type: image/png;
    name="Logo firma.png"
Content-Id: <2477BC4D-AE34-4C97-9B0A-****@Home>
```

This particular email was generated by Apple Mail client.

I also noticed that, if I remove the to_lf call in https://github.com/mikel/mail/blob/df48a05a7fb5a4271e6df12da7afb26a53494a18/lib/mail/encodings/quoted_printable.rb#L19, images would show up normally.

I plan to remove to_lf call but i'm thinking: Is there a reason why to_lf is called? Is it trying to fix something else?

**Comment by jeremy** [comment]
Quoted-printable is not binary-safe. It's… complicated 😊

Here's the backstory: https://github.com/mikel/mail/pull/496

We could make a stronger effort at "binary-safe" quoted-printable, but it's tough to make any guarantees, or even round-trip properly (!)

**Comment by fernandomm** [comment]
Just giving some feedback.

I've been running this gem without the .to_lf call ( only in decode method ) for almost one month. The software in which it's used process millions of emails per day and so far it didn't caused any issues.

All attachments in quoted printable work and it didn't affected emails with quoted printable content.

**Comment by jeremy** [comment]
Fixed by #1113
